packages/typescript-estree/tests/lib/persistentParse.ts

```diff
@@ -140,36 +142,6 @@ function baseTests(
     expect(() => parseFile('foo', PROJECT_DIR)).not.toThrow();
     expect(() => parseFile('bar', PROJECT_DIR)).not.toThrow();
   });
-
-  it('handles tsconfigs with no includes/excludes (single level)', () => {
-    const PROJECT_DIR = setup({}, false);
-
-    // parse once to: assert the config as correct, and to make sure the program is setup
-    expect(() => parseFile('foo', PROJECT_DIR)).not.toThrow();
-    expect(() => parseFile('bar', PROJECT_DIR)).toThrow();
-
-    // write a new file and attempt to parse it
-    writeFile(PROJECT_DIR, 'bar');
-
-    expect(() => parseFile('foo', PROJECT_DIR)).not.toThrow();
-    expect(() => parseFile('bar', PROJECT_DIR)).not.toThrow();
-  });
-
-  it('handles tsconfigs with no includes/excludes (nested)', () => {
-    const PROJECT_DIR = setup({}, false);
-
-    // parse once to: assert the config as correct, and to make sure the program is setup
-    expect(() => parseFile('foo', PROJECT_DIR)).not.toThrow();
-    expect(() => parseFile('baz/bar', PROJECT_DIR)).toThrow();
-
-    // write a new file and attempt to parse it
-    writeFile(PROJECT_DIR, 'baz/bar');
-
-    expect(() => parseFile('foo', PROJECT_DIR)).not.toThrow();
-    expect(() => parseFile('baz/bar', PROJECT_DIR)).not.toThrow();
-  });
-
-  // TODO - support the complex monorepo case with a tsconfig with no include/exclude
```

Comment:
I don't know why I put these inside the `baseTests`, they only apply to the `"tsconfig with no includes / files"` test case.